### ISSUE TITLE
Console oddity when starting Gramps running FamilyTreeView

### ISSUE THREAD

**Issue by emyoulation** [ztlxltl/FamilyTreeView#8]
I restarted Gramps (windows console with the the FamilyTreeView active after changing the Date display format.  It took at least 10 seconds for the Gramps GUI to appear after the console window. 

But the strange part was the message: "Why, when same code works in GraphView"

**Comment by PQYPLZXHGF** [comment]
> ...I'm using GrampsAIO. .... Or what do you mean by windows console?

Probably means @emyoulation started Gramps using the "GrampsAIOXX 5.2.3-console" shortcut icon in the Start menu. 

> Shortcuts During installation, one shortcut is placed on the desktop and two shortcuts are added the Start menu. 
> **Please only use the 'console' secondary shortcut when performing diagnostics requested by a developer.**
> 1) GrampsAIOXX 5.2.3       ← for normal use of Gramps
> **2) GrampsAIOXX 5.2.3-console**
> 
> https://gramps-project.org/wiki/index.php/Installing_Gramps_for_Windows_computers#All-In-One_installer

**Comment by emyoulation** [comment]
Yes. That's correct. Been tied up all day with other distractions. Sorry

**Comment by emyoulation** [comment]
A screenshot won't help. Is is just a console window with that exact text, not even a single character more ... before or after.

The 1st start of Gramps is always a bit longer. It is building a lot of __ pycache__ folders

**Comment by giotodibondone** [comment]
> But the strange part was the message: "Why, when same code works in GraphView"

Searching for those words only shows one addon has them on line [1241](https://github.com/gramps-project/addons-source/blob/maintenance/gramps52/PrerequisitesCheckerGramplet/PrerequisitesCheckerGramplet.py#L1241) 

>                print(_("Why, when same code works in Graphview"))

 in relation to testing for the presence of GooCanvas   it is the "Prerequisites Checker Gramplet" addon.

**Comment by fxtmtrsine** [comment]
@giotodibondone Well that explains that, saw this also, thanks for working it out 👍

@emyoulation maybe the message should be expanded to mention  "Prerequisites Checker Gramplet >  reports > Why, when same code works in GraphView"?

**Comment by emyoulation** [comment]
But report filed in MantisBT for PrerequisitesCheckerGramplet

[0013650](https://gramps-project.org/bugs/view.php?id=13650)	 	  	
2025-02-18	[PrerequisitesCheckerGramplet] message sent to console (for GooCanvas test) is not helpful
